.github/workflows/default.yml

```diff
@@ -15,14 +15,53 @@ jobs:
         uses: golangci/golangci-lint-action@v2
         with:
           version: v1.29
-  test:
-    name: test
+  unit-tests:
+    name: unit-tests
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
       - uses: actions/setup-go@v2
         with:
           go-version: "1.13"
 
-      - name: unit tests
+      - name: Run unit tests
         run: make test
+  acc-tests:
+    runs-on: ubuntu-latest
+    name: acc-tests
+    env:
+      REVIZOR_URL: ${{ secrets.REVIZOR_URL }}
+      REVIZOR_TOKEN: ${{ secrets.REVIZOR_TOKEN }}
+      SCALR_TOKEN: ${{ secrets.SCALR_TOKEN }}
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-go@v2
+        with:
+          go-version: "1.13"
+      - name: Environment settings
+        env:
+          API_BRANCH: ${{ contains(github.event.head_commit.message, '[API_BRANCH]') }}
+          DB_BRANCH: ${{ contains(github.event.head_commit.message, '[DB_BRANCH]') }}
+        run: |
+          echo "API_BRANCH=${API_BRANCH}" >> $GITHUB_ENV
+          echo "Set env variable from commit message: API_BRANCH=${API_BRANCH}"
+          echo "DB_BRANCH=${DB_BRANCH}" >> $GITHUB_ENV
+          echo "Set env variable from commit message: DB_BRANCH=${DB_BRANCH}"
+          echo BRANCH=${GITHUB_REF#refs/heads/} >> $GITHUB_ENV
+          echo "Set env variable: BRANCH=${GITHUB_REF#refs/heads/}"
+      - name: Create container
+        id: create
+        uses: Scalr/gh-action-revizor@feature/SCALRCORE-16917
+        with:
+          command: create
+      - name: Run acceptance tests
+        env:
+          SCALR_HOSTNAME: ${{ steps.create.outputs.hostname }}
+        run: make testacc
+      - name: Delete container
+        id: delete
+        if: ${{ steps.create.outputs.container_id }}
```

Comment:
Have you confirmed that this step runs even if the tests fail?

Reply:
Now it works.
See checks 36, 37
https://github.com/vmotso/check-gh-action-revizor/actions